### PR TITLE
update copyright

### PR DESCRIPTION
**Pull request by abhi16394**
# Description
Added copyrights to the go files that were missing it.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/491 |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by bharathsreekanth — APPROVED** [review]

File: base_test.go [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+Copyright © 2020 - 2022 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (satyakonduri) [inline base_test.go:2]: Can you please remove the spaces between the years `2020-2022` . We have followed the same structure for the other repositories.

Thank you

**Review by satyakonduri — CHANGES_REQUESTED** [review]

**Comment by shanmydell** [comment]
@abhi16394 : Update github id

**Comment by satyakonduri** [comment]
I see some of the `.sh` files and `Makefile` also don't have the copyright.
Can we add a copyright for those as well?
Thank you

**Review by spriya-m — CHANGES_REQUESTED** [review]
Please update Go version to 1.19

File: base_test.go (hunk 1 above)
Comment (abhi16394) [reply base_test.go]: Updated all the file with changes you requested

**Comment by abhi16394** [comment]
> @abhi16394 : Update github id



> I see some of the `.sh` files and `Makefile` also don't have the copyright. Can we add a copyright for those as well? Thank you

Added copyright to all sh files and makefiles usually dont have copyrights as i looked into other repositories.
so i skipped them.

File: internal/multipath/generate_mock.sh [hunk 2]
```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (satyakonduri) [inline internal/multipath/generate_mock.sh:1]: It should be 2020-2022 as we are modifying it now.

**Comment by abhi16394** [comment]
> Please update Go version to 1.19

updated go version

File: internal/powerpath/generate_mock.sh [hunk 3]
```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (satyakonduri) [inline internal/powerpath/generate_mock.sh:1]: It should be 2022 As we created this file in 2022.

File: internal/scsi/generate_mock.sh [hunk 4]
```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (satyakonduri) [inline internal/scsi/generate_mock.sh:1]: It should be 2020-2022 as we are modifying it now.

File: internal/wrappers/generate_mock.sh [hunk 5]
```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (satyakonduri) [inline internal/wrappers/generate_mock.sh:1]: It should be 2020-2022 as we are modifying it now.

**Review by satyakonduri — CHANGES_REQUESTED** [review]

File: internal/scsi/generate_mock.sh (hunk 4 above)
Comment (bharathsreekanth) [reply internal/scsi/generate_mock.sh:1]: @abhi16394 have all these comments been addressed?

Comment (bharathsreekanth) [reply internal/scsi/generate_mock.sh:1]: If yes, could you resolve them.

File: internal/wrappers/generate_mock.sh (hunk 5 above)
Comment (abhi16394) [reply internal/wrappers/generate_mock.sh]: changed it

File: internal/scsi/generate_mock.sh (hunk 4 above)
Comment (abhi16394) [reply internal/scsi/generate_mock.sh]: changed it

File: internal/powerpath/generate_mock.sh (hunk 3 above)
Comment (abhi16394) [reply internal/powerpath/generate_mock.sh]: changed it

File: internal/multipath/generate_mock.sh (hunk 2 above)
Comment (abhi16394) [reply internal/multipath/generate_mock.sh]: changed it

**Review by hoppea2 — APPROVED** [review]

**Review by satyakonduri — APPROVED** [review]

File: internal/wrappers/generate_mock.sh (hunk 5 above)
Comment (satyakonduri) [reply internal/wrappers/generate_mock.sh]: Thank you.

File: internal/scsi/generate_mock.sh (hunk 4 above)
Comment (satyakonduri) [reply internal/scsi/generate_mock.sh]: Thank you.

File: internal/powerpath/generate_mock.sh (hunk 3 above)
Comment (satyakonduri) [reply internal/powerpath/generate_mock.sh]: Thank you.

File: internal/multipath/generate_mock.sh (hunk 2 above)
Comment (satyakonduri) [reply internal/multipath/generate_mock.sh]: Thank you.

File: base_test.go (hunk 1 above)
Comment (satyakonduri) [reply base_test.go]: Thank you.

**Pull request merged**